localStorage.js

```diff
@@ -0,0 +1,26 @@
+const name1 = document.querySelector('#name');
```

Comment:
- [ ] I think there is an issue with the use of the `localStorage` because when I fill the contact form and reload the page it does not load any data, and actually, it does not save the data in the `localStorage`. Please, kindly address this issue in order to match the project requirements :wink: :raised_hands:
![image](https://user-images.githubusercontent.com/81522435/147250831-51cf87d1-2d49-468f-87d3-993922635809.png)

Reply:
Thank you. I just forget to put link to js file.